lib/utils/homePage.html

```diff
@@ -0,0 +1,63 @@
+<html>
+    <head>
+        <title>CORDOVA-PLUGIN-LIVERELOAD</title>
+        <style>
+            #loading, #error {
+                display: none;
+            }
+        </style>
+    </head>    
+    <body>
+        <h3>CORDOVA-PLUGIN-LIVERELOAD</h3>
+        <div id="error">There was an error connecting to the BrowserSync server.
+            <br />
+            <br />Try using the
+            <b>
+                <a>'tunnel'</a>
+            </b>option like so:
+            <br />
+            <b>`cordova run android -- --livereload --tunnel`</b>
```

Comment:
Suggestion: Try using the 'tunnel' option as in - 'cordova run android -- -- livereload --tunnel